frontend/src/bundles/common/components/radio/radio-group.tsx

```diff
@@ -1,50 +1,89 @@
-import { FormControlLabel, RadioGroup as MuiRadioGroup } from '@mui/material';
+import {
+    FormControlLabel,
```

Comment:
looks like we have this - https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/blob/2f3ff530a4486c3603a9ca616b747934bc054b62/frontend/src/bundles/common/components/components.ts#L6

Reply:
@nikita-remeslov  our custom FormLabel doesn't use FormControlLabel from mui, it uses FormLabel